notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1136_M1M3_analysis_stability_tracking.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #55.        seconds = tte_unix - tts_unix
</pre><p> </p><p>In this fuction, everything you have until here is repeating in the next function (detectUnstableEvents). In order to avoid duplicated code, I believe you could have an auxiliary function that execute those actions, that can be passed later to both computeSettleTrack and detectUnstableEvents (where you can really execute the steps that compute settle track and detect unstable events).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Are you suggesting that I change the code or take it into account for future analysis?</p>

Reply:
<p>I think it would be nice to have it in this notebook. It will make it easier to be read.</p>

Reply:
I have created a new function with all the part of the code that was repeated.